ictuwp-plugin-toolkit-taal.php

```diff
@@ -26,12 +26,17 @@
 // CONSTANTS:
 //========================================================================================================
 // `voorbeeld` custom post type
-defined( 'TT_EXAMPLE_CPT' )                         or define( 'TT_EXAMPLE_CPT', 'voorbeeld' );
+defined( 'TT_EXAMPLE_CPT' ) or define( 'TT_EXAMPLE_CPT', 'voorbeeld' );
 
 // `voorbeeld_type` taxonomy
-defined( 'TT_EXAMPLE_TYPE_TAX' )                    or define( 'TT_EXAMPLE_TYPE_TAX', 'voorbeeld_type' );
-defined( 'TT_EXAMPLE_TYPE_TAX_OVERVIEW_TEMPLATE' )  or define( 'TT_EXAMPLE_TYPE_TAX_OVERVIEW_TEMPLATE', 'template-voorbeeld_type-overview.php' );
-defined( 'TT_EXAMPLE_TYPE_TAX_DETAIL_TEMPLATE' )    or define( 'TT_EXAMPLE_TYPE_TAX_DETAIL_TEMPLATE', 'template-voorbeeld_type-detail.php' );
+defined( 'TT_EXAMPLE_TYPE_TAX' ) or define( 'TT_EXAMPLE_TYPE_TAX', 'voorbeeld_type' );
+defined( 'TT_PAGETEMPLATE_OVERVIEW_EXAMPLE_CPT' ) or define( 'TT_PAGETEMPLATE_OVERVIEW_EXAMPLE_CPT', 'template-voorbeelden.php' );
```

Comment:
Fijn dat de constanten niet meer verwijzen naar `EXAMPLE_TYPE_TAX` als we ze gebruiken voor de `EXAMPLE_CPT` 👍🏻

Kleinigheidje vwb consistente naamgeving:

Meestal is het in de vorm `<flavor>_<name>_<type>_<function>` (bv `TT_EXAMPLE_CPT` of `TT_EXAMPLE_TYPE_TAX`).

In die lijn klopt `TT_EXAMPLE_CPT_OVERVIEW_PAGEID` maar zou ik dus `TT_PAGETEMPLATE_OVERVIEW_EXAMPLE_CPT` veranderen naar `TT_EXAMPLE_CPT_OVERVIEW_TEMPLATE`

Reply:
Heb naamgeving herzien.
TT_PAGETEMPLATE_OVERVIEW_EXAMPLE_CPT > TT_EXAMPLE_CPT_OVERVIEW_TEMPLATE